shared/src/dateHelpers.ts

```diff
@@ -21,6 +21,12 @@ export const getCurrentDateInNewJersey = (): Dayjs => {
   return dayjs().tz("America/New_York");
 };
 
+export const getCurrentDateInNewJerseyISOFormatted = (): string => {
+  dayjs.extend(utc);
+  dayjs.extend(timezone);
+  return dayjs().tz("America/New_York").toISOString();
+};
```

Comment:
This is mostly covered by the helper function above. You could simplify this to be:

```typescript
export const getCurrentDateInNewJerseyISOFormatted = (): string => {
  return getCurrentDateInNewJersey().toISOString();
};
```